### DISCUSSION TITLE
Wiki update (dockerized postgres etc.)

### DISCUSSION THREAD

**Discussion by Plesoun** [bafolts/terraforming-mars#4953] [General]
Hi, 
I have been playing around a bit with the dockerized setup for my raspberry pi server. It includes docker-compose setup with postgres and nginx. I would like to update the wiki with how-to do this. I dont know what the process you have for this, can someone help? Thanks.

**Comment by kberg** [comment]
It looks like Github doesn't support PRs on wikis. However, you can clone the repo: https://github.com/terraforming-mars/terraforming-mars.wiki.git. If you're looking to update a single page, feel free to send it as an attachment to this discussion.

There are other ways to make the wiki editable by others (cloning the wiki and making it it's own first-class Github project) but I'm not prepared to take on the additional maintenance at this point in time.

**Reply by Plesoun** [reply]
Yea, I have cloned it created branch and tried to push, only then realizing it might not work like that with the git wikis :), anyway you are correct, it is an update of a single page -> see attatched :). Feel free to msg me if something is not clear, or if I left something out you deem important.
[Docker-Setup.md](https://github.com/terraforming-mars/terraforming-mars/files/9591313/Docker-Setup.md)

**Comment by kberg** [comment]
Applied. Thank you!

**Comment by kberg** [comment]
Closing this one. If you still need help, let me know.
